app/src/main/res/layout/settings_pill_with_subtitle.xml

```diff
@@ -23,27 +23,35 @@
     android:orientation="vertical"
     android:paddingBottom="10dp">
 
-    <LinearLayout
```

Comment:
Changed this to make sure if a text is very long here it will use more than one line and the beta pill will still shown.